bndtools.core/test/bndtools/wizards/repo/RepoBundleVersionToRangeTest.java

```diff
@@ -31,62 +33,81 @@
  * version increment)</li>
  * </ul>
  */
+@ExtendWith(SoftAssertionsExtension.class)
 public class RepoBundleVersionToRangeTest {
 
 	@Test
-	public void testVersionToVersionRanges() {
-		assertEquals(parseVersionRange("[1.2.3,1.2.4)").toString(),
-			toVersionRangeUpToNextMicro(Version.valueOf("1.2.3")).toString());
+	public void testVersionToVersionRanges(SoftAssertions softly) {
```

Comment:
Sorry to be painful... But I think using field injection rather than parameter injection might be more concise:
```java
@InjectSoftAssertions
SoftAssertions softly;
```
... then you can delete this parameter and it should still work.

Also, do you think you could squash this commit into the previous one?

Reply:
Done.